target/linux/ath79/dts/qca9558_airtight_c-65.dts

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/*
+ * This board has two identical flash ICs but only one is defined in this file.

```

Comment:
Remove this comment and place it into your commit message.

Reply:
Done.